website/src/docs/companion.md

```diff
@@ -181,6 +186,8 @@ export COMPANION_UPLOAD_URLS="http://master.tus.io/files/,https://master.tus.io/
 
 See [env.example.sh](https://github.com/transloadit/uppy/blob/master/env.example.sh) for an example configuration script.
 
+If a `...SECRET_FILE"` environment variable is defined, the according `secret` will be overridden by the file's content.
```

Comment:
I think this note should be added as a comment above the `COMPANION_AWS_SECRET_FILE` (and others) option itself. I fear a lot of users will find the `...SECRET_FILE"` notation confusing, and it also doesn't follow the current pattern of the documentation

Reply:
I tried to realize that in the recent commit. Is that what you meant?